errors.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+class ERR_INVALID_ARG_TYPE extends TypeError {
+  constructor(name, expected, actual) {
+    super(`${name} must be ${expected} got ${actual}`);
+  }
+}
+
+class ERR_NOT_IMPLEMENTED extends Error {
+  constructor(feature) {
+    super(`${feature} not implemented`);
+  }
+}
+
+module.exports = {
+  codes: {
```

Comment:
why are classes under "codes"? i'd expect "codes" to be an object of integers.

Reply:
This is a Nodeism, see:

https://github.com/nodejs/node/blob/5badf46f2a9363e6762900e18d4f85541738f738/lib/internal/streams/operators.js#L6